network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_blocks.html

```diff
@@ -3,7 +3,7 @@
     {% for profile in profiles %}
       <div class="col-md-6 col-12 mb-5">
 
-        <div class="person-card">
+        <div class="{% if profile.location %} person-card {% else %} person-card-sans-location {% endif %}">
```

Comment:
Same as above, why not `class="person-card{% if !profile.location %} no-location{% endif %}"`?